src/test/java/com/microsoft/aad/msal4j/AuthorityTest.java

```diff
@@ -169,4 +152,39 @@ public void testDoStaticInstanceDiscovery_ValidateFalse_TrustedAuthority()
         final AADAuthority aa = new AADAuthority(new URL(TestConfiguration.AAD_UNKNOWN_TENANT_ENDPOINT));
         //PS Assert.assertTrue(aa.doStaticInstanceDiscovery(false));
     }
+
+    @Test
+    public void testValidateAuthorityPath() {
+        String[] authoritiesWithInvalidPath = {
+                "https://login.microsoftonline.com/",
+                "https://login.microsoftonline.com//",
+                "https://login.microsoftonline.com//tenant",
+                "https://login.microsoftonline.com/tenant//",
+                "https://login.microsoftonline.com////tenant//path1"};
+
+        for (String testAuthority : authoritiesWithInvalidPath) {
+            try {
+                URL authorityUrl = new URL(testAuthority);
+                Authority.validateAuthority(authorityUrl);
+            } catch (Exception ex) {
+                Assert.assertTrue(ex instanceof IllegalArgumentException);
+                Assert.assertEquals
```

Comment:
Instead of having duplicated code below, can you add "https://login.microsoftonline.com" to this array and check if(testAuthority =="https://login.microsoftonline.com" ), then assert the specific error message.

Reply:
generally speaking test data which leads to different outcomes should be separated, refactored test